scripts/insert_into_appointments.sql

```diff
@@ -1 +1,29 @@
 -- insert appointments creating connections between doctors and patients
+
+INSERT INTO appointments(date, time, patient_id)
+  SELECT TIMESTAMP '2016-09-22 11:30', 20, d.id, p.id
+  FROM doctors d, patients p
+    WHERE d.specialty = 'Allergy and immunology'
+    AND d.given_name = 'Randal'
+    AND d.surname = 'Sloan'
+
+    AND p.given_name = 'Lela'
+    AND p.surname = 'Cannon'
+    AND p.born_on = '1980-10-22'
+  UNION
+  SELECT TIMESTAMP '2016-10-10  08:30', 20, d.id, p.id
+  FROM doctors d, patients p
+    WHERE d.specialty = 'Dermatology'
+    AND d.given_name = 'Helga'
+    AND d.surname = 'Akers'
+
+    AND p.given_name = 'Alba'
+    AND p.surname = 'Slater'
+    AND p.born_on = '1960-09-13'
+;
+
```

Comment:
a few problems here:
1) In the 1st line here, you have 3 parameters, date, time and patient_id. Look at your appointments table. How many columns do you have? 
2) where did you get `SELECT TIMESTAMP` from? what does this have to do with our appointments? 
3) following `SELECT TIMESTAMP`, you have 4 arguments, but you set this script up to accept 3 arguments.
4) the question asked for you to insert 4 specific appointments. The doctors you have here do not exist `in the data file. 

Here is one possible solution for a slightly different database setup:
```sql
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT '20160812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hilton'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20160817', '1300',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Talley')
  UNION
  SELECT '20160812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20160817', '1300',
   (SELECT id FROM doctors WHERE surname = 'Hilton'),
   (SELECT id FROM patients WHERE surname = 'Hodge')
;
```